easybuild/easyconfigs/a/Amber/Amber-16-iomkl-2016.07-AmberTools-16-patchlevel-5-14-serial.eb

```diff
@@ -0,0 +1,32 @@
+#
+# Based on https://raw.githubusercontent.com/hpcugent/easybuild-easyconfigs/master/easybuild/easyconfigs/a/Amber/Amber-14-intel-2016a-AmberTools-15-patchlevel-13-13.eb
+#
+name = 'Amber'
+version = '16'
+ambertools_ver = '16'
+# Patch levels from http://ambermd.org/bugfixes16.html and http://ambermd.org/bugfixesat.html
+patchlevels = (5, 14)  # (AmberTools, Amber)
+versionsuffix = '-AmberTools-%s-patchlevel-%s-%s-serial' %(ambertools_ver, patchlevels[0], patchlevels[1])
+
+homepage = 'http://ambermd.org/amber.html'
+description = """Amber (originally Assisted Model Building with Energy
+ Refinement) is software for performing molecular dynamics and structure
+ prediction."""
+
+toolchain = {'name': 'iomkl', 'version': '2016.07'}
+toolchainopts = {'usempi': False}
+
+sources = [
+    'Amber%(version)s.tar.bz2',
+    'AmberTools%s.tar.bz2' % ambertools_ver,
+]
+
+dependencies = [
+    ('netCDF', '4.4.0'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('Python', '2.7.11'),
+]
+
+patches = ['Amber-16_fix-hardcoding.patch']
```

Comment:
@edmondac please move this up below `sources`, and use `%(version)s` (avoid hardcoding the version in multiple places)